api/handlers/blockinfo_handler.go

```diff
@@ -771,6 +790,71 @@ func (h *BlockHandler) GetTxs() gin.HandlerFunc {
 	}
 }
 
+//Getdebts get all debts by order or by block
+func (h *BlockHandler) Getdebts() gin.HandlerFunc {
+	return func(c *gin.Context) {
+		dbClinet := h.DBClient
+
+		p, _ := strconv.ParseUint(c.Query("p"), 10, 64)
+		ps, _ := strconv.ParseUint(c.Query("ps"), 10, 64)
+		s, _ := strconv.ParseInt(c.Query("s"), 10, 64)
+		if ps == 0 {
+			ps = transItemNumsPrePage
+		} else if ps > maxItemNumsPrePage {
+			ps = maxItemNumsPrePage
+		}
+
+		if p >= 1 {
+			p--
```

Comment:
why --?
add comments here

Reply:
By the total number according to the article and ps distribution calculated totalPages, conversion
	if page > (totalPages - 1) {
		page = totalPages - 1
	}
	end = total - page*step